<launch>
    <!-- <arg name="model_file" default = "$(find semantic_slam)/model/darknet53_64/" /> -->
    <arg name="model_file" default = "$(find semantic_slam)/../../model/darknet53_64/" />

    <node pkg="semantic_slam"  type="aloam_semantic_node_loop"  name="aloam_semantic_node_loop"  output="screen">
        <param name="model_path" type="string" value="$(arg model_file)" />
        <param name="verbose" type="bool" value="true" />
        <param name="pose_topic" type="string" value="/Odometry" />
        <param name="lidar_topic" type="string" value="/kitti/velo/pointcloud" />
    </node>

    <node name="rviz_wind" pkg="rviz" type="rviz" args="-d $(find semantic_slam)/launch/loam_velodyne.rviz" />

</launch>